doc/classes/RenderingDevice.xml

```diff
@@ -29,6 +29,12 @@
 			<param index="2" name="size_bytes" type="int" />
 			<param index="3" name="post_barrier" type="int" enum="RenderingDevice.BarrierMask" is_bitfield="true" default="32767" />
 			<description>
+				Clears the contents of the [param buffer] ranging from [param offset] to [param size_bytes]. Always raises a memory barrier.
```

Comment:
```suggestion
				Clears the contents of the [param buffer], clearing  [param size_bytes] bytes, starting at [param offset]. Always raises a memory barrier.
```
My bad misread the original wording using "ranging" for a size

Reply:
Note: double space after "clearing"

Reply:
My bad! Need a laptop to save me from the foibles of autocorrect 🙃